modules/views/civicrm.views.inc

```diff
@@ -304,6 +304,9 @@ function civicrm_views_add_date_arguments(&$data, $value) {
  *   Array with the new custom field appended
  */
 function civicrm_views_custom_data_cache(&$data, $entity_type, $group_id, $sub_type) {
+  if (!CRM_Core_Config::isUpgradeMode()) {
```

Comment:
so i tested in upgrade and it made no difference i think that removing the ! will make a difference in non upgrade times